### ISSUE TITLE
Training hyper-parameters and pre-training

### ISSUE THREAD

**Issue by echatzo** [prithuls/MV-SWIN-T#5]
Greetings, 

I am trying to replicate your results but am currently not achieving the same performance.

Could you share the training hyper-parameters with more detail? 
What is the annealing strategy used ? Do you use warmup ? What are the ReduceLROnPlateau and early stopping parameters used ?

Furthermore, do you use any particular pre-training or weight initialization  not shown in the Github repo?

Thank you

**Comment by ayazmhmd** [comment]
I am facing the same issue. I am getting 56% f1 score with their implementation
